packages/apollo-codegen-swift/src/codeGeneration.ts

```diff
@@ -471,7 +471,7 @@ export class SwiftAPIGenerator extends SwiftGenerator<CompilerContext> {
 
         this.printNewlineIfNeeded();
         this.printOnNewline(
-          swift`public static let selections: [GraphQLSelection] = `
+          swift`public private(set) static var selections: [GraphQLSelection] = `
```

Comment:
I think that as long as the variable is static this will be retained in memory. What about
```suggestion
          swift`public var selections: [GraphQLSelection] { `
```

or maybe for old swift versions that don't support implicit return, which is a bit ugly as the line is wrong
```suggestion
          swift`public var selections: [GraphQLSelection] { return`
```

and close the `}` on line 481?

Reply:
Oof, yeah turning it into an instance var won't work because we need to be able to access it without an instance.

back to the drawing board...

Reply:
Also, feel free to ask on the issue how important this is for folks to have the right priority against the codegen rewrite work you're doing. As this looks to turn out to be a decent amount of work.

Reply:
Yeah...this was my procrastination project that somehow got away from me. 

However, I took a different approach on take 2, and it was WAY easier and still works. Yet another reminder to come back to stuff the next day when it's being dumb instead of just beating your head on the wall 🙃